src/Visualization/Python/PlotAlongLine.py

```diff
@@ -90,30 +83,18 @@ def points_on_line(
         " has an effect if multiple files are specified."
     ),
 )
-@click.option(
-    "--stylesheet",
-    "-s",
-    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
-    envvar="SPECTRE_MPL_STYLESHEET",
-    help=(
-        "Select a matplotlib stylesheet for customization of the plot, such "
-        "as linestyle cycles, linewidth, fontsize, legend, etc. "
-        "The stylesheet can also be set with the 'SPECTRE_MPL_STYLESHEET' "
-        "environment variable."
-    ),
-)
+@apply_stylesheet_command()
+@show_or_save_plot_command()
```

Comment:
Are the order of these two important? If so, we should not that somewhere

Reply:
No, not important